.gitlab-ci.yml

```diff
@@ -27,23 +24,42 @@ variables:
     helm --tiller-namespace ${KUBE_NAMESPACE} status "${DEPLOYMENT_NAME}"
 
 build:
+  cache:
```

Comment:
@JoseD92 Let's get rid of the cache for now and revisit setting up cache-s3 for the curator builds in a separate issue.

Reply:
We've discussed this and agreed that we will revisit it in a separate issue soon when we set up `cache-s3` for this